content/blog/posts/osu-story.rst

```diff
@@ -0,0 +1,15 @@
+OSL in the news!
+================
+:date: 2018-02-08
+:slug: osl-in-the-news
+:author: Jonathan Frederick
+:img: OSLWorkers2Adjusted.jpg
+:tag: featured-stories
+
+OSU has written a story about us! In the article they talk about the services
+we provide, our positive impact on the lives of our student workers, and the
+industries we work with.
+
+Check it out `here`_!
```

Comment:
I think I can just remove this and replace it with a link around the word "article" as I mentioned above.